resources/iam-role-policy.go

```diff
@@ -65,10 +68,17 @@ func ListIAMRolePolicies(sess *session.Session) ([]Resource, error) {
 	return resources, nil
 }
 
+func (e *IAMRolePolicy) Filter() error {
+	if strings.HasPrefix(aws.StringValue(e.role.Path), "/aws-service-role/") {
+		return fmt.Errorf("cannot alter service roles")
```

Comment:
why is this an error?

Reply:
Back then I thought it would be smart to design the interface like this ...

Reply:
Understood. Though we could log here and still return `nil`.

Reply:
That would not work, because we want the control loop to know that it should not attempt to delete this resource.